metadeploy/api/migrations/0098_use_org_id_consistently.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.16 on 2020-09-30 21:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0097_optional_preflight_user"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="job",
+            name="uuid",
+        ),
+        migrations.RemoveField(
+            model_name="preflightresult",
+            name="uuid",
+        ),
+        migrations.RemoveField(
+            model_name="scratchorgjob",
+            name="canceled_at",
+        ),
+        migrations.AddField(
+            model_name="scratchorgjob",
+            name="org_id",
+            field=models.CharField(blank=True, max_length=18, null=True),
+        ),
+        migrations.AlterField(
+            model_name="job",
+            name="org_id",
+            field=models.CharField(default="", max_length=18),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name="preflightresult",
+            name="org_id",
+            field=models.CharField(default="", max_length=18),
+            preserve_default=False,
```

Comment:
I wasn't sure what to do about these defaults. I guess we could leave these nullable -- it'll just break things if they're missing, so I wasn't sure why they should be optional. 🤷